runtime/src/app/app-routing.module.ts

```diff
@@ -12,14 +10,18 @@ const routes: Routes = [
     pathMatch: 'full'
   },
   {
-    path: 'plan/list',
-    loadChildren: './components/plan-list.module#PlanListModule'
-  },
-  {
     path: 'plan/board',
     loadChildren: './components/plan-board.module#PlanBoardModule'
   },
   {
+    path: 'plan/detail',
+    loadChildren: './components/plan-detail.module#PlanDetailModule'
+  },
+  {
+    path: 'plan/list',
+    loadChildren: './components/plan-list.module#PlanListModule'
+  },
+  {
```

Comment:
Any particular reason why list is after detail? Should it not be before or after board?

Reply:
Alphabetical order or board > detail > list